core/unix/signal.c

```diff
@@ -3671,7 +3671,6 @@ convert_frame_to_nonrt(dcontext_t *dcontext, int sig, sigframe_rt_t *f_old,
     f_new->sig_noclobber = f_new->sig;
```

Comment:
Please describe in the PR description text box (turns into the merge commit message) how this was tested.